django/db/migrations/recorder.py

```diff
@@ -69,13 +69,16 @@ def ensure_schema(self):
             raise MigrationSchemaMissing("Unable to create the django_migrations table (%s)" % exc)
 
     def applied_migrations(self):
-        """Return a set of (app, name) of applied migrations."""
+        """Return a dict of (app, name), Migration of applied migrations."""
```

Comment:
Maybe "Return a dict of (app, name) to Migration instance of applied migrations."

(Just not sure the comma reads...)

Reply:
Works for me.